packages/design-system/src/components/web-components/ds-choice/__snapshots__/ds-choice.test.tsx.snap

```diff
@@ -132,6 +139,12 @@ exports[`Choice is a checkbox 1`] = `
         >
           George Washington
         </label>
+        <p
+          aria-atomic="true"
+          aria-live="assertive"
+          class="ds-c-inline-error"
+          id="undefined__error"
```

Comment:
This is going to be duplicated in the DOM pretty quickly, so I don't think we can ship it in this state. Do you know why this is undefined?

Reply:
Looks like it's actually happening on the React Choice in Storybook too

Reply:
This is some thing I was seeing in PROD right now. I think you can see it in the web component side of things.

Reply:
Okay, I see it in prod now

Reply:
[Created a fix for it](https://github.com/CMSgov/design-system/pull/3146)